lib/wings/active_fedora_converter.rb

```diff
@@ -34,7 +34,7 @@ def initialize(resource:)
     end
 
     ##
-    # Accesses and parses the attributes from the resource through ConverterValueMapper
+    # Accesses and parses the attributes from the resource through ConvetrerValueMapper
```

Comment:
Typo?

Reply:
i suppose so, but not one introduced in this PR.

Reply:
why is GH showing it as a change then?

Reply:
oh huh. i assumed it was a whitespace change, but i guess i must have done a stealth letter transpose in my editor.

sorry for the confusion, and thanks for the catch!